Actions/AL-Go-Helper.ps1

```diff
@@ -2216,3 +2230,34 @@ function RetryCommand {
         }
     }
 }
+
+function GetProjectsFromRepository {
+    Param(
+        [string] $baseFolder,
+        [string[]] $projectsFromSettings,
+        [string] $selectProjects = ''
+    )
+    if ($projectsFromSettings) {
+        $projects = $projectsFromSettings
+    }
+    else {
+        # For multiple projects, get all folders in two levels below the base folder containing an .AL-Go folder with a settings.json file
```

Comment:
This function is really just because that exact same code for locating projects in a repo existed in 4 places: CheckForUpdates, Deliver, DetermineProjectsToBuild and IncrementVersionNumber - so collecting the code in one place seemed like a good idea.
Changing functionality to allow to have AL-Go projects in any depth seems like a risky change as part of this.

Reply:
Ok, let's keep it as is then.